index.html

```diff
@@ -15,14 +15,27 @@
             <a href="#" class="logo">
                 <p>Bonjour</p>
             </a>
-            <div><img src="./images/Union@2x.png" alt="icon menu" id="menu">
+            <div id="hamburger-menu">
+                <button id="hamburger" type="button" class="hamburger">
+                    <span class="ham-top"></span>
+                    <span class="ham-mid"></span>
+                    <span class="ham-bottom"></span>
+                </button>
             </div>
```

Comment:
- [ ] Great job so far 👍🏻 I can see that on mobile size,  the hamburger menu button is visible 👏🏻 however it does not work 🤔 According to the project requirements, when the user clicks (or taps) the hamburger button, the mobile menu should appear. Kindly address this 💯 😉  and remember:

1. When the user clicks on the close (X) button, the mobile menu should disappear.
1. When the user clicks on any of the mobile menu options, the correct part of the page should display.
2. When the user clicks on any option of the mobile menu, the mobile menu should disappear.
3. On desktop size make sure the mobile navigation menu page with the close button and the hamburger menu button is not visible.

Reply:
@BenMukebo Hi there. On my side it is perfectly fine and all the functionality is working fine.
can you please check it again

Reply:
@BenMukebo I guess maybe if you have not checked out to the mobile-menu branch, then it will probably won't work.
One must checkout to mobile-menu branch to get it worked properly.

Reply:
@Zuhaib042, sorry for the late reply, I can see that you fixed the issue and the project got approved, congrats🥳 🎉